polls.sql

```diff
@@ -0,0 +1,43 @@
+DROP TABLE if EXISTS creators cascade;
+DROP TABLE if EXISTS polls cascade;
+DROP TABLE if EXISTS options cascade;
+DROP TABLE if EXISTS responses cascade;
+
+CREATE TABLE polls(
+   poll_id varchar(100) primary key not null,
+    question text not null,
```

Comment:
indentation here tells me you're not using a style prettifyer in your IDE - so have a look around and install one

Reply:
fixed :)